frontend/src/components/common/ToastMessage.vue

```diff
@@ -5,11 +5,6 @@ import Toast from 'primevue/toast';
 import { useToast } from 'primevue/usetoast';
 import { IconSize } from '@/enum/IconEnum';
 
-import {
-    useNotificationMessage,
-    useErrorDialog,
```

Comment:
Hi Nick, since we are not using the Dialog to display error anymore, let's check if we still need this "useErrorDialog" in state, or we can remove it completely. We might can also remove the Dialog component in the Summary page.

Reply:
I have removed because it is not used anymore.